### DISCUSSION TITLE
VaporFile Issue while uploading large files

### DISCUSSION THREAD

**Discussion by prakhargvp** [laravel/nova-issues#5951] [Q&A]
- Laravel Version: 10.9
- Nova Version: 4.24.4
- PHP Version: 8.2
- Database Driver & Version: 
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

I'm getting 504 gateway timeout error while saving the resource.

![CleanShot 2023-10-10 at 13 54 08](https://github.com/laravel/nova-issues/assets/7056772/e5ab0184-a865-4b9f-b21d-5ef8c84a0129)
![CleanShot 2023-10-10 at 13 20 06](https://github.com/laravel/nova-issues/assets/7056772/ef88a3b3-a6e3-4312-b2c6-4805fbd79297)
![CleanShot 2023-10-10 at 10 48 17](https://github.com/laravel/nova-issues/assets/7056772/e5f15875-e511-41ed-8b9d-0537241e3e77)
![CleanShot 2023-10-10 at 10 46 18](https://github.com/laravel/nova-issues/assets/7056772/5d23293d-fec1-4fed-9511-f7509407743b)

**Comment by crynobone** [comment]
5xx error suggest an issue with the server, might be best to check against Laravel Vapor logs and ensure configuration are suitable for such usage.
